presto-docs/src/main/sphinx/release/release-319.rst

```diff
@@ -0,0 +1,45 @@
+===========
+Release 319
+===========
+
+General Changes
+---------------
+* Fix planning failure for queries involving ``UNION`` and ``DISTINCT`` aggregates. (:issue:`1510`)
+* Fix excessive runtime when parsing expressions involving ``CASE``. (:issue:`1407`)
```

Comment:
Sounds maybe like an improvement rather than a fix?

Reply:
It's in that gray area where it's so slow that it's effectively a bug (it's exponential, so it can take "forever")